pyquil/paulis.py

```diff
@@ -78,10 +78,12 @@ def id(self):
             self._id = s
             return s
 
+    def __eq__(self, other):
+        return self.id() == other.id()
+
     def __len__(self):
-        """l
-        The length of the pauliterm is the number of Pauli operators in the term. This is
-        equivalent to the length of self._ops dict
+        """
+        The length of the pauliterm is the number of Pauli operators in the term.
```

Comment:
great catch. this should return a length 1 as the smallest length for any pauliterm. have updated to fix